src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,52 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.0
+
+**New features**
+
+* Added a new `subctl uninstall` command that removes all Submariner components and dataplane artifacts, such as IP table rules, routes from
+a cluster.
+* Added a new `subctl unexport` command that stops exporting a previously exported service.
+* Added new `subctl cloud prepare` and `subctl cloud cleanup` commands for the Red Hat OpenStack Platform (RHOS).
+* New Globalnet metrics were added:
+  * Count of global Egress IPs allocated at Cluster scope, namespace scope and for selected pods per CIDR.
+  * Count of global Ingress IPs allocated for Pods/Services per CIDR.
+* Added support for Globalnet objects verification using subctl diagnose command.
+* Added support for `--broker-namespace` flag while deploying the Broker.
+* Added support for running `subctl diagnose` on single node clusters.
+* Added support for running `subctl diagnose` from a pod in cluster.
+* `subctl cloud prepare` now deploys a dedicated gateway node as a default option on GCP and OpenStack platforms.
```

Comment:
I'm wondering if the K8s docs guidance to avoid "starting a sentence with a component tool or component name" applies here.

https://kubernetes.io/docs/contribute/style/style-guide/#starting-a-sentence-with-a-component-tool-or-component-name

I'm okay with keeping it as-is because it's easy to parse visually.